src/style.css

```diff
@@ -1 +1,36 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-right: 4px;
```

Comment:
Remove margin-right from the last stars in a row (use :last-child rule)
![image](https://github.com/mate-academy/layout_stars/assets/80700247/814655a4-f11c-4fb4-9e5f-eaccdfc52bfc)